investment-app/TRINV.Api/TRINV.Application/Commands/CreateIdentityUserCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace TRINV.Application.Commands;
+
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using System.Text.Json;
+using MediatR;
+
+public record CreateIdentityUserCommand : IRequest
+{
+    [Required]
+    [StringLength(20, ErrorMessage = "The Username must be between 3 and 20 characters long.", MinimumLength = 3)]
+    public string Username { get; set; }
+
+    [Required]
+    [StringLength(100, ErrorMessage = "The Password must be between 6 and 100 characters long.", MinimumLength = 6)]
+    public string Password { get; set; }
+
+    [Required]
+    [EmailAddress]
+    public string Email { get; set; }
+}
+
+public class CreateUserCommandHandler : IRequestHandler<CreateIdentityUserCommand>
```

Comment:
The handler name should be CreateIdentityUserCommandHandler !

Reply:
Done.